### ISSUE TITLE
Pytorch 1.12.0 onnx export update broke GPT-2 support by TensorRT 8.4.1

### ISSUE THREAD

**Issue by pommedeterresautee** [NVIDIA/TensorRT#2119]
## Description

The last update of Pytorch (1.12.0) has changed the way models are exported on ONNX, aka the graph is quite different from what you get from Pytorch 1.11

It appeared that GPT2 model exported to ONNX is not anymore supported by TensorRT 8.4.1
It seems to be related to the dynamic axis, indeed, if I made them static, `trtexec` doesn't complain anymore.
Moreover, everything works fine with dynamic axis and Pytorch 1.11

## Environment

**TensorRT Version**: 8.4.1
**NVIDIA GPU**: 3090 RTX
**NVIDIA Driver Version**: 515.48.07
**CUDA Version**: 11.7
**CUDNN Version**: 8.4.1
**Operating System**: Ubuntu 22.04
**Python Version (if applicable)**: 3.9
**Tensorflow Version (if applicable)**: N/A 
**PyTorch Version (if applicable)**: 1.12.0
**Baremetal or Container (if so, version)**: baremetal

## Relevant Files

Files can be generated from Hugging Face transformer library.

## Steps To Reproduce

On terminal:

```bash
pip install torch --extra-index-url https://download.pytorch.org/whl/cu113
pip install transformers[onnx]
python -m transformers.onnx --model=gpt2  ./
/usr/src/tensorrt/bin/trtexec --onnx=./model.onnx --verbose 
```

It will output something like that:

```log
[07/04/2022-09:52:19] [E] [TRT] ModelImporter.cpp:773: While parsing node number 616 [Slice -> "onnx::Slice_657"]:
[07/04/2022-09:52:19] [E] [TRT] ModelImporter.cpp:774: --- Begin node ---
[07/04/2022-09:52:19] [E] [TRT] ModelImporter.cpp:775: input: "h.1.attn.bias"
input: "onnx::Slice_651"
input: "onnx::Slice_653"
input: "onnx::Slice_3738"
input: "onnx::Slice_656"
output: "onnx::Slice_657"
name: "Slice_616"
op_type: "Slice"

[07/04/2022-09:52:19] [E] [TRT] ModelImporter.cpp:776: --- End node ---
[07/04/2022-09:52:19] [E] [TRT] ModelImporter.cpp:779: ERROR: builtin_op_importers.cpp:4067 In function importSlice:
[8] Assertion failed: (axes.allValuesKnown()) && "This version of TensorRT does not support dynamic axes."
[07/04/2022-09:52:19] [E] Failed to parse onnx file
[07/04/2022-09:52:19] [I] Finish parsing network model
[07/04/2022-09:52:19] [E] Parsing model failed
[07/04/2022-09:52:19] [E] Failed to create engine from model or file.
[07/04/2022-09:52:19] [E] Engine set up failed
&&&& FAILED TensorRT.trtexec [TensorRT v8401] # /usr/src/tensorrt/bin/trtexec --onnx=./model.onnx --verbose
```

Complete verbose logs:
[error.log.zip](https://github.com/NVIDIA/TensorRT/files/9037450/error.log.zip)

**Comment by nvpohanh** [comment]
Could you try const-folding the ONNX file and see if the error goes away? https://github.com/NVIDIA/TensorRT/tree/main/tools/Polygraphy/examples/cli/surgeon/02_folding_constants

Thanks

**Comment by pommedeterresautee** [comment]
It worked!
Do you think next version of TRT will manage it directly?

Please find logs below:
[error.log.zip](https://github.com/NVIDIA/TensorRT/files/9038076/error.log.zip)

**Comment by ttyio** [comment]
closing since no activity for more than 14 days, please reopen if you still have question, thanks!
